w08h03/test/pgdp/datastructures/UnitTest.java

```diff
@@ -97,7 +98,7 @@ private String generateString(Random rnd) {
 	public void parallelIteratorsTest() {
 		QuarternarySearchTree<Integer> t = new QuarternarySearchTree<>();
 
-		Random random = new Random();
+		Random random = new Random(seed);
```

Comment:
Thanks for adding the seed to `Random` 👍 However, can you revert this change because we are going to remove this test (see #174) and this would result a merge conflict.

Reply:
done

Reply:
thanks 👍